functions/Get-DbaHideInstance.ps1

```diff
@@ -0,0 +1,140 @@
+function Get-DbaHideInstance {
+    <#
+    .SYNOPSIS
+        Gets Hide Instance settings for a SQL Server instance
+
+    .DESCRIPTION
+        Gets Hide Instance settings for a SQL Server instance. Note that this requires access to the Windows Server - not the SQL instance itself.
+
+        This setting is found in Configuration Manager.
+
+    .PARAMETER SqlInstance
+       The target SQL Server instance or instances. Defaults to localhost.
+
+    .PARAMETER Credential
+        Allows you to login to the computer (not sql instance) using alternative Windows credentials
+
+    .PARAMETER EnableException
+        By default, when something goes wrong we try to catch it, interpret it and give you a friendly warning message.
+        This avoids overwhelming you with "sea of red" exceptions, but is inconvenient because it basically disables advanced scripting.
+        Using this switch turns this "nice by default" feature off and enables you to catch exceptions with your own try/catch.
+
+    .NOTES
+        Tags:   HideInstance, Security
```

Comment:
Just tag as `Instance, Security`

Will make it more discoverable.

Reply:
The tag in the other commends Enable and Disable is HideInstance, so I thought for consistency it would be best to have the same, I could be wrong.

Reply:
Ah, yeah sorry. I am going to be cleaning all those up, just have not gotten to it.

If you want to make them all consistent in this PR that would be awesome!

Reply:
Ok, all three are the same, I like consistency.